docs/docs_source/wopi/files/RefreshLock.rst

```diff
@@ -14,17 +14,35 @@ RefreshLock
     The refreshed lock must expire automatically after 30 minutes unless it is modified by a subsequent WOPI
     operation, such as :ref:`Unlock` or :ref:`RefreshLock`.
 
+    ..  include:: /fragments/priorlock.rst
+
+    ..  include:: /fragments/lock409.rst
+
+    ..  include:: /fragments/no_lock_id.rst
+
+    ..  note::
+        Since there is no GetLock operation defined in WOPI, Office Online will sometimes make a RefreshLock call to
```

Comment:
You could make it more generic in case this changes in the future. "...Office Online will sometimes make lock requests to the host..." I guess now that I write it saying lock request is specific since we have that request.